ci-operator/config/openshift/jenkins-sync-plugin/openshift-jenkins-sync-plugin-master.yaml

```
@@ -35,6 +35,18 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-jenkins
+- as: e2e-aws-operator
```

Nit on name - since these are e2e tests from within the plugin repo, I would call it `e2e-aws-jenkins-sync-plugin`

```suggestion
- as: e2e-aws-jenkins-sync-plugin
```